modules/utxo-lib/test/bitgo/litecoin/fixtures/ltc-mweb-transaction.json

```diff
@@ -0,0 +1 @@
+"02000000000801c9acff2c66910bba2faca1b7c6889f5f69e2ac04ff42452ccbb05e84d1dd55f90000000000ffffffff0287cec866e9000000225820bce2cc319d7393fdb728489d1c3060b129b70ef20304af59d88d0def48669e63100905000000000017a9143c006dfa10642aa0e7f7171c2a3afb78f7529e96870000000000"
```

Comment:
Find MWEB ltc transactions by looking [on the explorer](https://www.mwebexplorer.com/blocks) by finding blocks that have MWEB transactions